website/src/components/events/event-list-item.js

```diff
@@ -85,25 +85,35 @@ export default function EventListItem(props) {
     eventType = "Community Event"
   }
 
+  // Gatsby seems to hijack all <a> tags and, if they do not begin with http://
+  // or https://, appends the path to the domain instead of setting the URL as
+  function checkLink(param) {
+    const linkElements = ["http", "twitch", "www", ".ca", ".com"]
+
+    if (!param.includes(" ") && linkElements.some(e => param.includes(e))) {
+      if (!param.includes("http")) param = "http://" + param
+      return <Typography variant="a" component={Link} to={param}>{param}</Typography>;
```

Comment:
I'm getting an error in the console running your branch locally, and I think it might be because this component isn't expecting a variant with the value "a". For now, let's skip the fancy MaterialUI stuff and just go for a simple `<a href={param} target="_blank" rel="noreferrer">{param}</a>`

Reply:
it's now a simple `a` tag as above